### PR TITLE
Remove travis configuration file

### PR DESCRIPTION
**Pull request by jasontedor**
This commit removes the Travis configuration file as it's out-of-date.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +0,0 @@
-language: java
-jdk:
-  - openjdk7
```
Comment (jasontedor) [inline .travis.yml]: This line here leads me to believe that it is not being used at all since master has long required JDK 8.

**Comment by pickypg** [comment]
LGTM

**Comment by nik9000** [comment]
LGTM. I tried to get travis working a long, long time ago but it was ultimately fruitless. The build just takes too long for it to work well with travis.

**Comment by jasontedor** [comment]
Thanks @pickypg and @nik9000!

**Pull request merged**